addons/website_sale/static/src/js/product_configurator_dialog/product_configurator_dialog.js

```diff
@@ -59,4 +56,8 @@ patch(ProductConfiguratorDialog.prototype, {
     showShopButtons() {
         return this.props.isFrontend && !this.props.edit;
     },
+      get formattedTotal() {
```

Comment:
```suggestion

    get formattedTotal() {
```
Ident also it would be better to add little doc string :see_no_evil: